### PR TITLE
feature(unlock-app): adding link to manage lock page

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding a link to the event page for organizers to manage their attendees.

# Issues

Refs #12238

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Screenshot 2023-06-26 at 7 22 29 PM

**Review by searchableguy — APPROVED** [review]

File: unlock-app/src/components/content/event/EventDetails.tsx [hunk 1]
```diff
@@ -714,6 +714,31 @@ export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
...
+                    title="Manage Attendees"
+                    description="See who is attending your event, invite people with airdrops and more!"
+                  />
+                </div>
+                <div className="md:col-span-1">
+                  {eventLocks?.map(({ lockAddress, network }) => {
+                    return (
+                      <Button
+                        key={lockAddress}
+                        as="a"
```
Comment (searchableguy) [inline unlock-app/src/components/content/event/EventDetails.tsx:729]: Probably use next link here since it's a relative link

**Review by searchableguy — COMMENTED** [review]
One comment

File: unlock-app/src/components/content/event/EventDetails.tsx (hunk 1 above)
Comment (julien51) [reply unlock-app/src/components/content/event/EventDetails.tsx:729]: Ha! ok let me try!

**Pull request merged**